### PR TITLE
adding DroppedLinksCount to non-otlp exporters section

### LINKED ISSUE #1696: Links limit handling not defined in exporter spec
The section in the SDK exporter spec about reporting dropped attributes and events based on span limits isn't mentioning links. It seems like it should.

https://github.com/open-telemetry/opentelemetry-specification/blob/0d02e95ed403152e88b1d0896eeb67394b78ed62/specification/trace/sdk_exporters/non-otlp.md#dropped-attributes-count

### PR DESCRIPTION
**Pull request by codeboten**
Fixes #1696 

## Changes

Adding a similar section to events and attributes limit to the exporter specification.
Related issues https://github.com/open-telemetry/opentelemetry-specification/issues/1656

### CODE REVIEW

File: specification/trace/sdk_exporters/jaeger.md [hunk 1]
```diff
@@ -22,22 +22,23 @@ See also:
...
-| Span.TraceId             | Span.traceIdLow/High | Span.trace_id | See [IDs](#ids)     |
-| Span.ParentId            | Span.parentSpanId | as SpanReference | See [Parent ID](#parent-id)     |
-| Span.SpanId              | Span.spanId       | Span.span_id     |      |
-| Span.TraceState          | TBD               | TBD              |      |
-| Span.Name                | Span.operationName | Span.operation_name |  |
-| Span.Kind                | Span.tags["span.kind"] | same | See [SpanKind](#spankind) for values mapping |
-| Span.StartTime           | Span.startTime | Span.start_time | See [Unit of time](#unit-of-time) |
-| Span.EndTime             | Span.duration | same | Calculated as EndTime - StartTime. See also [Unit of time](#unit-of-time) |
-| Span.Attributes          | Span.tags | same | See [Attributes](#attributes) for data types for the mapping.            |
+| OpenTelemetry              | Jaeger Thrift    | Jaeger Proto     | Notes |
```
Comment (carlosalberto) [inline specification/trace/sdk_exporters/jaeger.md:25]: Are the style changes in both tables really needed? They seemed fine before, albeit with one space less ;)

**Review by carlosalberto — APPROVED** [review]

File: specification/trace/sdk_exporters/jaeger.md (hunk 1 above)
Comment (codeboten) [reply specification/trace/sdk_exporters/jaeger.md:25]: it's only my preference to keep things aligned in the table, i find it easier to read the markdown this way. I can change it back if you prefer.

**Review by arminru — APPROVED** [review]
Thanks!
Please also mention this in `CHANGELOG.md`.

**Comment by codeboten** [comment]
Thanks @arminru, changelog updated.

**Pull request merged**